packages/spark-core/components/_accordions.scss

```diff
@@ -76,14 +70,30 @@ $accordion-max-width: 53.125rem;
   color: $blue-dark;
 }
 
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary--open .sprk-c-Accordion__heading,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary:hover .sprk-c-Accordion__heading,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary:active .sprk-c-Accordion__heading,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary:focus .sprk-c-Accordion__heading,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary--open .sprk-c-Accordion__icon,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary:hover .sprk-c-Accordion__icon,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary:active .sprk-c-Accordion__icon,
-.sprk-c-Accordion--navigation .sprk-c-Accordion__summary:focus .sprk-c-Accordion__icon {
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary--open
+.sprk-c-Accordion__heading,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary:hover
+.sprk-c-Accordion__heading,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary:active
+.sprk-c-Accordion__heading,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary:focus
+.sprk-c-Accordion__heading,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary--open
+.sprk-c-Accordion__icon,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary:hover
+.sprk-c-Accordion__icon,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary:active
+.sprk-c-Accordion__icon,
+.sprk-c-Accordion--navigation
+.sprk-c-Accordion__summary:focus
+.sprk-c-Accordion__icon {
```

Comment:
What's going on here?

Reply:
@yodasw16 , it's some styles that @RCopeland  added for navigation. Could use Sass nesting of the states.